content/en/tracing/trace_retention_and_ingestion/_index.md

```diff
@@ -111,6 +112,32 @@ To configure for 100% ingestion on every service instrumented with a Datadog tra
 DD_TRACE_SAMPLE_RATE=1.0
 ```
 
+### Ingestion Breakdown
+
+The ingestion breakdown column breaks down the destination of all traces originating from the service. It can help you explain lower than expected ingestion rates and missing traces.
+
+The breakdown is composed of the following parts:
+
+**Complete traces ingested (Green)**:
+
+Shows you the percentage of traces that have been ingested by Datadog.
+
+**Complete traces not retained (Grey)**:
+
+Shows you the percentage of traces that have purposefully not been forwarded to Datadog by the agent or the tracer.
+
+This can happen for one of two reasons depending on your configuration:
+1. By default, the agent and the tracers intelligently sets the service ingestion rate. See [Change the Default Ingestion Rate](#change-the-default-ingestion-rate) if you wish to configure this behavior.
+2. When you change the default ingestion rate to less than 100%.
+
+**Complete traces dropped by the tracer rate limiter (Orange)**:
+
+When you choose to [configure the ingestion rate of a service](#change-the-default-ingestion-rate), you explicitly define the ingestion rate that your service should have. However, as a protection mechanism, a rate limiter set to 100 traces/second by default gets automatically enabled. If you wish to configure this rate limiter, please [open a support ticket][5] so we can guide you through the process.
+
+**Traces dropped due to the agent CPU limit (Red)**:
+
+The agent has a configuration option allowing users to limit the usage of the CPU. After this limit is reached the agent will stop accepting traces from the tracers. Refer to the agent configuration to [configure how much CPU](https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config_template.yaml#L736-L741) you wish to allocate to the agent.
```

Comment:
```suggestion
- **Complete traces ingested** (green): The percentage of traces that have been ingested by Datadog.
- **Complete traces not retained** (gray): The percentage of traces that have intentionally not been forwarded to Datadog by the agent or the tracer. This can happen for one of two reasons depending on your configuration:

    1. By default, the agent and the tracers intelligently set the service ingestion rate. See [Change the Default Ingestion Rate](#change-the-default-ingestion-rate) to configure this behavior.
    2. When you change the default ingestion rate to less than 100%.

- **Complete traces dropped by the tracer rate limiter** (orange): When you choose to [configure the ingestion rate of a service](#change-the-default-ingestion-rate), you explicitly define the ingestion rate that your service should have. However, as a protection mechanism, a rate limiter set to 100 traces per second by default is automatically enabled. To configure this rate limiter, [open a support ticket][5] so we can guide you through the process.

- **Traces dropped due to the agent CPU limit** (red): The agent has a configuration option allowing users to limit the usage of the CPU. After this limit is reached the agent will stop accepting traces from the tracers. Change the [agent configuration][9] to configure how much CPU to allocate to the agent.
```

Reply:
and add 
```
[9]: https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config_template.yaml#L736-L741
```
at the end of the file